apps/common/src/main/scala/org/lfdecentralizedtrust/splice/config/AutomationConfig.scala

```diff
@@ -27,12 +27,12 @@ case class AutomationConfig(
     pollingJitter: Double = 0.2,
     /** Enabled schedling reward operations unfiromly across the first tick of a round opening.
       */
-    enableNewRewardTriggerScheduling: Boolean = true,
+    enableNewRewardTriggerScheduling: Boolean = false,
```

Comment:
so you also have no confidence in the ReceiveFaucetCouponTriggers? Those seemed less risky to me.